docs/ntp.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_label: Configure NTP
+title: ''
```

Comment:
Is this supposed to be empty?

Reply:
yep, we have it empty for all the other sections...  my guess is that it is never used 🤔